.ui/library/lviPerson.cpx

```diff
@@ -25,15 +23,63 @@
 			}
 		},
 		{
-			"className": ".sf-label .sf-listViewItem-simple-title",
+			"className": ".sf-flexLayout .sf-listViewItem-simple .sf-listViewItem-inactive .sf-listViewItem-active",
+			"id": "1fef-4b87-213d-4951",
+			"props": {
+				"children": [
+					"55f5-f296-a37b-fc7f",
+					"24ca-f21f-dfb6-5dc9",
+					"8474-887d-3102-405e",
+					"bd8f-5e00-4110-7fd7"
+				],
+				"name": "flPersonWrapper",
+				"parent": "1841-ff9a-b968-028d",
+				"usePageVariable": true
+			},
+			"source": {
+				"page": "__library__"
+			},
+			"type": "FlexLayout",
+			"userProps": {
+				"flexProps": {
+					"flexGrow": 1
+				},
+				"testId": "ph7qCLXQM",
+				"usePageVariable": true
+			}
+		},
+		{
+			"className": ".sf-imageView",
+			"id": "55f5-f296-a37b-fc7f",
+			"props": {
+				"children": [],
+				"name": "ivPerson",
+				"parent": "1fef-4b87-213d-4951",
+				"usePageVariable": true
+			},
+			"source": {
+				"page": "__library__"
+			},
+			"type": "ImageView",
+			"userProps": {
+				"flexProps": {},
+				"height": 52.27272727272727,
```

Comment:
On ImageView, it is better to specify the dimensions on a different class rather than user properties